apps/web/components/booking/pages/AvailabilityPage.tsx

```diff
@@ -15,6 +15,10 @@ import {
 import { EventType } from "@prisma/client";
 import * as Collapsible from "@radix-ui/react-collapsible";
 import { useContracts } from "contexts/contractsContext";
+import dayjs from "dayjs";
+import customParseFormat from "dayjs/plugin/customParseFormat";
+import utc from "dayjs/plugin/utc";
+import MarkdownIt from "markdown-it";
```

Comment:
<img width="487" alt="image" src="https://user-images.githubusercontent.com/3504472/179087292-d79da6d0-ad6b-41a0-b612-4232e638d4ba.png">

Can we convert markdown to html on the server? I'd rather not add unnecessary bloat to the client.